park_api/geodata.py

```diff
@@ -41,6 +41,17 @@ def __init__(self, city):
                 self._process_json(json.load(f))
         except FileNotFoundError:
             self.lots = {}
+        private_file = city[:-3] + ".json"
```

Comment:
Ah, I see. We should probably make that a little more obvious outside of this PR^^